tests/Integration/Behaviour/Features/Context/Domain/Product/ProductImageFeatureContext.php

```diff
@@ -328,24 +317,6 @@ public function assertProductHasNoImages(string $productReference): void
         );
     }
 
-    /**
-     * @Then product :productReference should have no images for shops :shops
-     *
-     * @param string $productReference
-     * @param string $shopReferences
-     */
-    public function assertProductHasNoImagesForShops(string $productReference, string $shopReferences): void
```

Comment:
Why did you decide to remove this test I'm genuinely curious The implementation didn't reflect the actual query behaviour it's true, but I believe the images were filtered programmatically so that this endpoint still made sense

Reply:
it wasn't used anymore. And as now we get images for all shops always (except the associated shop ids differs) we can just check if product has no images at all (in any shop)